tfbpmodeling/__main__.py

```diff
@@ -615,7 +623,7 @@ def test_sigmoid_interactor_significance(
 
     model_vars = topn_sig_coefs.keys()
 
-    results = evaluate_interactor_significance(
+    results = evaluate_interactor_significance_linear(
```

Comment:
Why is this changed to evaluate_interactive_significance_linear when the point is to be able to toggle between the two options? Shouldn't this be evaluate_interactor_significance?

```suggestion
    results = evaluate_interactor_significance(
```